nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/admin/SetHazelcastConfiguration.java

```diff
@@ -201,6 +201,9 @@
     @Param(name = "kubernetesServiceName", optional = true, alias = "kubernetesservicename")
     private String kubernetesServiceName;
 
+    @Param(name = "enableAutoPortSelection", optional = true, defaultValue = "true")
```

Comment:
I wouldn't have a default value of "true" here.
It means that you always have to specify it if you want to keep it disabled.

So if I did this:
`asadmin set-hazelcast-configuration --enableAutoPortSelection false --lite true`
followed by this:
`asadmin set-hazelcast-configuration --memberName wibbles`
the end result would be that the auto-port selection would still be enabled.

Reply:
Agreed. Had not thought about this.